src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Windows.cs

```diff
@@ -527,7 +527,7 @@ private static ProcessInfo[] GetProcessInfos(PerformanceCounterLib library, int
                                 // at the end.  If instanceName ends in ".", ".e", or ".ex" we remove it.
                                 if (instanceName.Length == 15)
                                 {
-                                    if (instanceName.EndsWith(".", StringComparison.Ordinal))
+                                    if (instanceName.EndsWith("."))
```

Comment:
Is it possible to use char '.' instead of the single-char string here as well?

Reply:
No.
https://github.com/dotnet/runtime/pull/81831#discussion_r1100365621